sbt/sbt-openapi-codegen/src/main/scala/codegen/BasicGenerator.scala

```diff
@@ -0,0 +1,190 @@
+package codegen
+
+import codegen.openapi.models.OpenapiModels.{OpenapiDocument, OpenapiParameter, OpenapiPath, OpenapiRequestBody, OpenapiResponse, OpenapiResponseContent}
+import codegen.openapi.models.OpenapiSchemaType
+import codegen.openapi.models.OpenapiSchemaType.{OpenapiSchemaArray, OpenapiSchemaBoolean, OpenapiSchemaDouble, OpenapiSchemaFloat, OpenapiSchemaInt, OpenapiSchemaLong, OpenapiSchemaObject, OpenapiSchemaRef, OpenapiSchemaSimpleType, OpenapiSchemaString}
+
+object BasicGenerator {
+
+  def generateObjects(doc: OpenapiDocument) = {
+    s"""|
+        |$packageStr
+        |
+        |object $obbjName {
+        |
+        |${indent(2)(imports)}
+        |
+        |${indent(2)(classDefs(doc))}
+        |
+        |${indent(2)(endpointDefs(doc))}
+        |
+        |}
+        |""".stripMargin
+  }
+
+  def packageStr: String = "package sttp.tapir.generated"
```

Comment:
going forward we'd probably want the package name to be configurable via a setting?

Reply:
Definitely!